angular1/README.md

```diff
@@ -0,0 +1,57 @@
+# Angular Input Mask
+
+## Getting started
+
+First, install it.
+
+```bash
+npm i angular1-text-mask --save
+```
+
+Then, use it as follows:
+
+```html
+<script
+  type="text/javascript"
+  src="./node_modules/angular1-text-mask/dist/angular1TextMask.js"></script>
```

Comment:
Is there an easy way to verify if this line will actually work? I mean if the script is included like the line above, will `angular.module('app', ['text-mask'])` work?